### DISCUSSION TITLE
How to use persist with immer?

### DISCUSSION THREAD

**Discussion by Athrun-Judah** [pmndrs/zustand#2173] [Q&A]
I'm trying to use both persist and immer, but ts reports an error: Argument of type 'StateCreator<StoreState, [],[["zustand/persist",unknown],["zustand/immer",never]]>' is not assignable to parameter of type 'StateCreator<StoreState,[],[never, unknown][]>'.

**Comment by dbritto-dev** [comment]
@Athrun-Judah please use 4.4.4 for now until we fix it
